build.psm1

```diff
@@ -696,19 +690,12 @@ function New-PSOptions {
     }
 
     if (-not $Runtime) {
-        $RuntimeFromDotNet = dotnet --info | ForEach-Object {
+        $Runtime = dotnet --info | ForEach-Object {
             if ($_ -match "RID") {
                 $_ -split "\s+" | Select-Object -Last 1
```

Comment:
I have 2 options, first do the conversion here. Other is to make runtime parameter mandatory. I prefer the first one. So if user does not specify -runtime we auto-select win7-x86/x64 on windows. On Linux/Mac, we don't change anything. What do you think?